### DISCUSSION TITLE
在iOS的Safari浏览器中点击容器端口无法跳转

### DISCUSSION THREAD

**Discussion by 001ProMax** [1Panel-dev/1Panel#6641] [Q&A]
### 1Panel 版本

v1.10.18-lts

### 问题描述

在iOS的Safari浏览器中点击容器端口无法跳转

### 重现步骤

1、使用iOS的Safari浏览器打开1Panel
2、点击容器端口，无法跳转

### 期待的正确结果

载新标签页打开对应端口

**Comment by wanghe-fit2cloud** [comment]
建议先检查是否是 Safari 版本问题导致的，你也可以开启 Safari 开发者模式，查看控制台和网络请求是否有异常。
我在本地环境使用 v1.10.18-lts 版本测试时未复现该问题，见下图：
<img width="1474" alt="image" src="https://github.com/user-attachments/assets/6d929962-32c8-4a58-9159-f26ad993d744">

> 也尝试更换一下 Chrome 或者 Firefox 浏览器再打开容器端口。

**Reply by 001ProMax** [reply]
Mac上的Safari没问题，但在iOS Safari上打开的1panel面板没法实现跳转
